cg/meta/workflow/microsalt.py

```diff
@@ -258,16 +258,17 @@ def get_deliverables_to_store(self) -> list:
         deliverables_to_store = []
         for case_object in self.db.cases_to_store(pipeline="microbial"):
             deliverables_file = self.get_deliverables_file_path(order_id=case_object.name)
-            if Path(deliverables_file).exists():
-                deliverables_to_store.append(deliverables_file)
+            if not Path(deliverables_file).exists():
```

Comment:
```suggestion
            if not deliverables_file.exists():
```
No need to convert to Path since the function now return Path objects